bluetooth_mesh/cli/gatt_client.py

```diff
@@ -100,7 +102,27 @@ def _mesh_beacon_received(self, _, payload):
             self._network.beacon_receive(beacon, auth)
 
     def _network_pdu_received(self, _, payload):
-        pass
+        iv_index, ctl, ttl, seq, src, dst, transport_pdu= NetworkMessage.unpack(payload, 
+                                        self._network.network_keys, self._network.iv_index)
+        if transport_pdu:
+            if src >= 0x8000 or dst == 0:
+                return
+            for node in self._network.nodes.values():
+                for ele in node.elements:
+                    if ele['unicastAddress'] == src:
+                        if (iv_index << 24 | seq) <= (self._network.iv_index << 24 | ele["sequenceNumber"]):
+                            return
+                        ele["sequenceNumber"] = seq
+            key = (src, dst, seq)
+            if key in self._access_message_rxed.keys():
```

Comment:
`msg = self._access_message_rxed.setdefault(key, AccessMessage(src, dst, ttl, None))`

Also, I think `_access_message_rxed` could be a set, not a dict.

Reply:
thanks for the suggestion, i really love the shorthand coding.

a dict would help to find the `AccessMessage` quickly, and here the main purpose is to handle messages from diffrent source concurrently. Also, a timer could be used to remove invalid `AccessMessage` in this dict.

Reply:
Right. I keep forgetting that python's `set` doesn't have lookup API